src/app/api/uploadgar/validations.js

```diff
@@ -48,9 +48,11 @@ module.exports.validations = (voyageObj, crewArr, passengersArr) => {
     [new ValidationRule(validator.notEmpty, '', voyageObj.departureTime, 'Enter a value for the departure time')],
     [new ValidationRule(validator.notEmpty, '', voyageObj.arrivalDate, 'Enter a value for the arrival date')],
     [new ValidationRule(validator.notEmpty, '', voyageObj.departureDate, 'Enter a value for the departure date')],
+    [new ValidationRule(validator.dateNotInPast, '', voyageObj.departureDate, 'Please enter current or future departure date')],
+    [new ValidationRule(validator.dateNotInPast, '', voyageObj.arrivalDate, 'Arrival date must atleast be today and cannot be more than 1 month in the future')],
```

Comment:
Can we reuse the same error messages as dateNotTooFarInFuture? Lines 51 and 52 could use'field_departure_date_too_far_in_future' and 'field_arrival_date_too_far_in_future'

Reply:
done